src/shim/runtime/v2/rune/types/types.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+const (
+	// PodSandbox identifies an infra container that will be used to create the pod
+	PodSandbox = "sandbox"
+	// PodContainer identifies a container that should be associated with an existing pod
+	PodContainer = "container"
+
+	// Default identifies the default namespace used by objects
+	Default = "default"
+	// KubeSystem identifies the namespace used by the Kubernetes system objects
+	KubeSystem = "kube-system"
```

Comment:
The values might be clearer if they had a prefix or suffix (`DefaultNS` and `KubeSystemNS`, for example).